### ISSUE TITLE
WARNING: The class 'ServerRoute' is not key value coding-compliant for the key 'arch'

### ISSUE THREAD

**Issue by cpg** [amahi/ios#76]
Run-time warning:

```
WARNING: The class 'ServerRoute' is not key value coding-compliant for the key 'arch'
 There is no support for optional type, array of optionals or enum properties.
As a workaround you can implement the function 'setValue forUndefinedKey' for this. See the unit tests for more information
```

**Comment by codedentwickler** [comment]
@cpg I looked into this issue, the warning is coming from EVReflection, It tries to convert every property in the JSON object to s swift type. If some property isn't defined in the swift class that exist in JSON then EVReflection prints this warning.

Ways to fix

1. Ensure all properties in the JSON object are mapped correctly to a swift type, i.e no JSON property should be ignored.

2. We can add a dictionary of all the keys to be ignored in the JSON response. This means EVReflection won't warn us anymore.

Which of the option do you find more appropriate. I prefer option 1.

**Comment by csoni111** [comment]
I would prefer the 2nd option. ie. we should ignore the keys which are not required. Why waste memory in storing values for not required keys!

**Comment by codedentwickler** [comment]
Thanks, @csoni111. Unnecessary memory usage is a good reason, you convinced me. I will proceed once @cpg sign off on this. Earlier I thought we might need those values later on but then we can always remove them from the ignored list if we need them.

**Comment by codedentwickler** [comment]
This bug has been fixed in #79
